tsconfig.json

```diff
@@ -8,10 +8,12 @@
         "noImplicitThis": true,
         "inlineSourceMap": false,
         "sourceMap": true,
+        "rootDir": "./src",
```

Comment:
What was the problem you are solving by adding this and the below `"include": ["src/**/*"]`.

Reply:
If you don't limit it to the src folder it'll pull in the ts files from the ui folder and cause issues compiling. One possible alternative would be to do: 
```
"exclude": ["ui/**/*"]
```
@prajwalkshetty might know of some reason that that wouldn't work as he did most of this base setup.